core/flyout.js

```diff
@@ -209,46 +250,74 @@ Blockly.Flyout.prototype.getMetrics_ = function() {
     // Flyout is hidden.
     return null;
   }
-  var viewHeight = this.height_ - 2 * this.SCROLLBAR_PADDING;
-  var viewWidth = this.width_;
+
   try {
     var optionBox = this.workspace_.getCanvas().getBBox();
   } catch (e) {
     // Firefox has trouble with hidden elements (Bug 528969).
-    var optionBox = {height: 0, y: 0};
+    var optionBox = {height: 0, y: 0, width: 0, x: 0};
+  }
+
+  var absoluteTop = this.SCROLLBAR_PADDING;
```

Comment:
I know it was before, so no objections, but do you know why is absoluteTop= scrollbar_padding in the vertical case? I was expecting 0 or some constant unrelated to scrolling. Something like Flyout.MARGIN or Flyout.CORNER_RADIUS